### DISCUSSION TITLE
Welcome to ClusterGVis Discussions!

### DISCUSSION THREAD

**Discussion by junjunlab** [junjunlab/ClusterGVis#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bigcatgan** [comment]
你好，我在使用getClusters(exp = dat)的时候出现了报错：
**Error in do_one(nmeth) : 外接函数调用时不能有NA/NaN/Inf(arg1)
此外: Warning messages:
1: In stats::dist(x) : 强制改变过程中产生了NA
2: In storage.mode(x) <- "double" : 强制改变过程中产生了NA**
导入的数据是两两比较后的log2fc值，同时把各处理中不显著的log2fc换成了0.001，不知道这种情况怎么解决
> dat<-read.csv("时间点时间序列分析.csv")
> head(dat,10)
           gene_id  PvsN PCvsNC    PLvsNL
1  ENSRNA050013833 0.001  0.001 -1.652759
2  ENSRNA050013835 0.001  0.001 -1.306907
3  ENSRNA050013836 0.001  0.001 -1.654837
4  ENSRNA050013837 0.001  0.001 -1.259903
5  ENSRNA050013843 0.001  0.001 -1.744778
6  ENSRNA050013844 0.001  0.001 -1.350752
7  ENSRNA050013845 0.001  0.001 -1.637795
8  ENSRNA050013849 0.001  0.001 -1.640271
9  ENSRNA050013851 0.001  0.001 -1.242763
10 ENSRNA050013854 0.001  0.001 -1.229870
